app/services/publish/public_desc_metadata_service.rb

```diff
@@ -3,16 +3,17 @@
 module Publish
   # Creates the descriptive XML that we display on purl.stanford.edu
   class PublicDescMetadataService
-    attr_reader :cocina_object, :constituents
+    attr_reader :cocina_object, :constituents, :include_access_conditions
 
     MODS_NS = 'http://www.loc.gov/mods/v3'
 
     # @param [Cocina::Models::Collection,Cocina::Models::DRO] cocina_object
     # @param [Array<Hash>] constituents a list of constituents (virtual object members) that are part of this object
-    def initialize(cocina_object, constituents)
+    def initialize(cocina_object, constituents, include_access_conditions: true)
```

Comment:
Can you please add the doc for the new parameter?

Reply:
I would love to, if only I knew what it meant or why it was used. The `#ng_xml` method it came from didn't document that information.